src/fides/api/schemas/connection_configuration/connection_secrets_rds_mysql.py

```diff
@@ -0,0 +1,26 @@
+from pydantic import Field
+
+from fides.api.schemas.base_class import NoValidationSchema
+from fides.api.schemas.connection_configuration.connection_secrets_base_aws import (
+    BaseAWSSchema,
+)
+
+
+class RDSMySQLSchema(BaseAWSSchema):
+    """
+    Schema to validate the secrets needed to connect to a RDS MySQL Database
+    """
+
+    username: str = Field(
+        default="fides_explorer",
```

Comment:
nit: `fides_explorer` feels a bit arbitrary/random to me as a default, we don't use "explorer" anywhere else in the product and it's not really a standard. I'd probably just stick with `fides` or `fides_service_user` or `fides_db_user` or something like that